src/MeshInertiaCalculator.hh

```diff
@@ -114,6 +124,10 @@ namespace gz
         public: std::optional<gz::math::Inertiald> operator()(
           sdf::Errors& _errors,
           const sdf::CustomInertiaCalcProperties& _calculatorParams);
+
+        /// \brief Percentage error tolerance allowed when testing if principal
+        /// moments of a mass matrix satify the triangle inequality.
+        public: static constexpr double kPrincipalMomentPercentTol = 0.05;
```

Comment:
```suggestion
        public: constexpr double kPrincipalMomentRelativeTol = 0.05;
```

suggest name change

also, does it need to be `static`?

Reply:
renamed variable and moved it outside of class so it does not need to be static. [a7ba9bf](https://github.com/gazebosim/gz-sim/pull/2775/commits/a7ba9bf7baf4baef65802d3a9b6f0931449a84b5)